internal/core/config/filters.go

```diff
@@ -40,8 +40,12 @@ func (mf *MetricFilter) Normalize() (map[string][]string, error) {
 	dimSet := map[string][]string{}
 	for k, v := range mf.Dimensions {
 		switch s := v.(type) {
-		case []string:
-			dimSet[k] = s
+		case []interface{}:
+			newSet := []string{}
+			for _, iv := range s {
+				newSet = append(newSet, fmt.Sprintf("%v", iv))
```

Comment:
Can you not check of s.(type) is string? Or is it actually a non-string value?

Reply:
If somebody gives an integer, for example, I'll just take it and convert to a string rather than trying to worry about validating it.